turbopack/crates/turbo-tasks-backend/src/backend/operation/mod.rs

```diff
@@ -455,7 +455,6 @@ impl<B: BackingStorage> TaskGuard for TaskGuardImpl<'_, B> {
         self.task_id
     }
 
-    #[must_use]
```

Comment:
@sokra looks like a new lint in the compiler detected that this has no effect:

```
warning: `#[must_use]` has no effect when applied to a provided trait method
   --> turbopack/crates/turbo-tasks-backend/src/backend/operation/mod.rs:458:5
    |
458 |     #[must_use]
    |     ^^^^^^^^^^^
    |
    = note: `#[warn(unused_attributes)]` on by default
```

Reply:
Can we move it to the `trait TaskGuard` declaration? Does it work then?